pkg/controllers/kube-scheduler.go

```diff
@@ -60,8 +60,10 @@ func (s *KubeScheduler) configure(cfg *config.MicroshiftConfig) {
 		klog.Fatalf("initialization error command options: %v", err)
 	}
 
+	klog.Infof("kube-scheduler config: %s", cfg.ControlPlane.KubeSchedulerCustomConfigPath)
+
 	args := []string{
-		"--config=" + cfg.DataDir + "/resources/kube-scheduler/config/config.yaml",
+		"--config=" + cfg.ControlPlane.KubeSchedulerCustomConfigPath,
```

Comment:
I think it's probably good enough for now, I wonder if we may need to merge what we configure in that config before passing it down to kube-scheduler (for example, to inject the kubeconfig file or setup leader election etc...)

Reply:
Maybe.  I agree that we should provide a more intelligent "override" path.  Whether MicroShift should do that by default, or via a discrete config/flag I'm not sure yet.